src/app/location/Station.js

```diff
@@ -141,13 +142,15 @@ define([
                 this.fGroup = new L.FeatureGroup().addTo(this.vMap.map);
                 this.pointDef.setMap(this.vMap.map, this.fGroup);
 
-                var that = this;
-                this.vMap.map.on('moveend', function () {
```

Comment:
It was the source of a bug. The map extent of the main map is set on successful station submit but I will set it when the dialog is closed as well.